### ISSUE TITLE
Are input images normalized?

### ISSUE THREAD

**Issue by zungam** [matterport/Mask_RCNN#791]
Is normalization or some similar algorithm a pre-processing step on images fed into mask_rcnn? If not, do you think this will make my model perform bad on transfer learning due to new images being very matte?

**Comment by jimmy15923** [comment]
Hi! I think it already used RGB subtract for image normalization, which is same normalized strategy as pre-train ResNet. You can find code in model.py

```python
def mold_image(images, config):
    """Expects an RGB image (or array of images) and subtraces
    the mean pixel and converts it to float. Expects image
    colors in RGB order.
    """
    return images.astype(np.float32) - config.MEAN_PIXEL
```

**Comment by rbavery** [comment]
From this issue https://github.com/matterport/Mask_RCNN/issues/95 it looks like this is only the correct normalization strategy for pretrained COCO, not imagenet. So I think for imagenet you would need to change the band order of your inputs and change `mold_image` to also divide by variance.

**Comment by zungam** [comment]
Thankyou @rbavery  and @jimmy15923  :)
